src/uct/api/uct.h

```diff
@@ -1142,21 +1142,22 @@ struct uct_iface_attr {
      * interface, this would usually be a combination of device and system
      * characteristics and determined at run time.
      */
-    double                   overhead;     /**< Message overhead, seconds */
-    uct_ppn_bandwidth_t      bandwidth;    /**< Bandwidth model */
-    ucs_linear_func_t        latency;      /**< Latency as function of number of
-                                                active endpoints */
-    uint8_t                  priority;     /**< Priority of device */
-    size_t                   max_num_eps;  /**< Maximum number of endpoints */
-    unsigned                 dev_num_paths;/**< How many network paths can be
-                                                utilized on the device used by
-                                                this interface for optimal
-                                                performance. Endpoints that connect
-                                                to the same remote address but use
-                                                different paths can potentially
-                                                achieve higher total bandwidth
-                                                compared to using only a single
-                                                endpoint. */
+    double                   overhead;      /**< Message overhead, seconds */
+    uct_ppn_bandwidth_t      bandwidth;     /**< Bandwidth model */
+    ucs_linear_func_t        latency;       /**< Latency as function of number of
+                                                 active endpoints */
+    double                   memory_latency;/**< NUMA memory latency */
```

Comment:
Leftovers. Will be remove.
We can store the memory latency on the worker iface to reduce calls to `topo_get_memory_distance`, but I don't think that here is the right place.
Let me know what you think.